testnet/start-obscuro-node.sh

```diff
@@ -3,8 +3,9 @@
 #
 # This script downloads and builds the obscuro node
 #
-# Note: Be aware that a network MUST always have AT LEAST ONE Genesis node -> Flag is_genesis=true
-#       Otherwise you might see your node getting stuck in waiting for a secret
+# Note: Be aware that a network MUST always have:
+#  * EXACTLY ONE genesis node -> Flag is_genesis=true (otherwise your node wil spin forever waiting for the network secret)
+#  * AT LEAST ONE aggregator  -> Flag node_type=aggregator (otherwise no rollups will be produced)
```

Comment:
maybe add a note here that these the single aggregator must also be the genesis for now.